flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/options/JdbcOptions.java

```diff
@@ -100,14 +100,20 @@ public Builder setUsername(String username) {
 			return this;
 		}
 
+		/**
+		 * optional, CONNECTION_CHECK_TIMEOUT_SECONDS.
+		 */
+		public Builder setCONNECTION_CHECK_TIMEOUT_SECONDS(int setCONNECTION_CHECK_TIMEOUT_SECONDS) {
```

Comment:
Please keep the parameters named with camel case.

Reply:
> Please keep the parameters named with camel case.

thanks，i will keep it now